onyo/commands/tree.py

```diff
@@ -53,7 +54,8 @@ def tree(args, opdir: str) -> None:
     orig_cwd = os.getcwd()
     try:
         os.chdir(opdir)
-        status = os.system(f"tree {dirs}")
+        ret = subprocess.run(['tree'] + dirs, capture_output=True, text=True)
```

Comment:
This should use `subprocess`'s `cwd`. Then `os` can be removed entirely, and the `chdir` dance can be avoided.